gui/network/migrations/0006_globalconfiguration_gc_domains.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-23 11:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('network', '0005_vlan_pcp.py')
```

Comment:
Did you manually fiddle with that like, it should not have ended with .py, this is leading to the error:
django.db.migrations.exceptions.NodeNotFoundError: Migration network.0006_globalconfiguration_gc_domains dependencies reference nonexistent parent node ('network', '0005_vlan_pcp.py')